repaygithub/cactus
# CACTUS-463 :: Modal Fixes

Ticket: https://repayonline.atlassian.net/browse/CACTUS-463

### Testing
1. Change the modal's content to "This is a Modalaslkdfjlaskdjflkasdjflkasjdflkajdlkfjasldkfjlaskdjflkasjdflkj" & make sure the content does not overflow.
2. Set the modal's height/maxHeight using the knobs & make sure the values you set are being respected.
